buildlib/azure-pipelines-release.yml

```diff
@@ -54,22 +52,26 @@ stages:
               set -eEx
               echo "Get commit message target $(system.pullRequest.sourceCommitId)"
               title=`git log -1 --format="%s" $(system.pullRequest.sourceCommitId)`
+              run_by_title=No
               if [[ "$title" == "AZP/RELEASE: "* ]]
               then
-                  echo Yes
-                  echo "##vso[task.setvariable variable=Title;isOutput=true]Yes"
+                  run_by_title=Yes
+              fi
+
+              if [[ ( "$(Build.Reason)" == "IndividualCI" ) || ( "${run_by_title}" == "Yes" && "$(Build.Reason)" == "PullRequest" ) ]]
```

Comment:
why need ( ) ?

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch

syntax

Reply:
or on ( ) || ( ) in conditional , fo clear understand

Reply:
( ) creates sub-shell, so in general IMO better use like this
`if [[ "$(Build.Reason)" == "IndividualCI" ]] || [[ "${run_by_title}" == "Yes" && "$(Build.Reason)" == "PullRequest" ]]`
but since it's back port, it's not so critical